content/projects/gsoc/2020/projects/github-checks.adoc

```diff
@@ -24,4 +23,48 @@ links:
   idea: /projects/gsoc/2020/project-ideas/github-checks
 ---
 
-// TODO: add details here, remove `showGoogleDoc: true`
+The link:https://developer.github.com/v3/checks/[GitHub Checks API] allows developers to report the CI integrations’ detail information rather than the binary pass/fail build status on GitHub pages. This project is about implementing this API as a new Jenkins plugin. By consuming this API, other plugins can easily create GitHub checks. Thus, any information during the Jenkins CI process like warnings, summaries, and durations can be directly shown on GitHub pages.
+
+=== Project Deliverables
+* A new Jenkins plugin that implements GitHub Checks API 
+* Pipeline supprt feature for GitHub Checks API
+* A new feature for Warnings Next Generation plugin to report warnings to GitHub by consuming the API
```

Comment:
```suggestion
* A new feature for plugin:warnings-ng[Warnings Next Generation plugin] to report warnings to GitHub by consuming the API
```

It would be nice to hyperlink components to simplify navigation. This macro will create a link to https://plugins.jenkins.io/warnings-ng/

Reply:
These predefined macros are cool!